sale_exception/tests/__init__.py

```diff
@@ -1,4 +1 @@
-# (c) 2015 Oihane Crucelaegui - AvanzOSC
-# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
```

Comment:
License should be here but copyright shouldn't, see https://github.com/OCA/maintainer-tools/tree/master/template/module/

Reply:
Fixed